### ISSUE TITLE
Goood old Mr.CraigTunstall Dirt.

### ISSUE THREAD

**Issue by Mka0207** [MrCraigTunstall/zombiesurvival#39]
https://www.dropbox.com/s/b65frywmgf40ebw/CraigExposed_Final.zip?dl=0

**Comment by MrCraigTunstall** [comment]
If I recall Zombine is part of the HL2. Nothing stolen. So again please respectfully leave.

**Comment by Mka0207** [comment]
The Zombine's model and sounds are yes however I created the code used to make that class unique in ZS. That's not why I am here, you will find out why I am here pretty soon.

**Comment by MrCraigTunstall** [comment]
I'll be waiting. If you haven't got a real issue to post then kinda leave, thanks.

**Comment by Mka0207** [comment]
![image](https://user-images.githubusercontent.com/7522605/38769835-e867e3ce-3fce-11e8-8570-130527cbf3ca.png)

**Comment by Mka0207** [comment]
;)

**Comment by MrCraigTunstall** [comment]
Yep that's when we both were working on it. UnderYouFive did it. All this over a logo? I mean if it's that bad I can simply get a new one.

**Comment by MrCraigTunstall** [comment]
Wait, I don't even have that logo added.

**Comment by Mka0207** [comment]
![nngwen1](https://user-images.githubusercontent.com/7522605/38769870-92517a4e-3fcf-11e8-9a6f-1b2848af926c.png)

**Comment by Mka0207** [comment]
I just don't see why you think adding my one of my workshop addons in and not giving credit is ok.

**Comment by MrCraigTunstall** [comment]
Yeah that's a meme that was posted when we were working on it. A simple joke at the time

**Comment by Mka0207** [comment]
I haven't done shit to you and you seemed to have jumped to Damien's protection after he started attacking FWKZT for no real reason.

**Comment by MrCraigTunstall** [comment]
Fine, you want credit added? Is that what you want? Let's talk this out Mka.

**Comment by Mka0207** [comment]
Ok fine, discord

**Comment by MrCraigTunstall** [comment]
Wait whoa? No no I stay on no ones side. I'm just here to host mine and play on servers. I checked his out like once.

**Comment by Mka0207** [comment]
Ok.

**Comment by MrCraigTunstall** [comment]
We can talk on that, mind giving me a few hours? Got guests round

**Comment by Mka0207** [comment]
Yes.
